modules/serverless-enabling-tls-internal-traffic.adoc

```diff
@@ -23,15 +23,15 @@ include::snippets/technology-preview.adoc[]
 
 .Procedure
 
-. Create or update your `KnativeServing` resource and make sure that it includes the `internal-encryption: "true"` field in the spec:
+. Create or update your `KnativeServing` resource and make sure that it includes the `system-internal-tls: Enabled` field in the spec:
```

Comment:
These changes are a separate ticket, so not sure if they are just here to be merged together?

Reply:
@ReToCode Per Martin's suggestion, this section is getting removed due to those new transport encryption docs, with two steps from this section preserved: https://github.com/openshift/openshift-docs/pull/82435/commits/361a5c167989bae5689c59c9dd3881d7007352aa